AppProject/AppProject/ChosenItemViewController.swift

```diff
@@ -0,0 +1,279 @@
+        label.frame = CGRect(x: 0, y: 0, width: 220, height: 20)
+        label.center = CGPoint(x: view.center.x, y: 570)
+        label.text = Constants.deviceConnectionText
+        label.textAlignment = .center
+        label.font = UIFont.systemFont(ofSize: 9, weight: .medium)
+        label.textColor = .systemGray2
+        return label
+    }()
+    
+    private lazy var addToCartButton: UIButton = {
+        let button = UIButton()
+        button.frame = CGRect(x: 0, y: 0, width: view.bounds.width - 30, height: 45)
+        button.center = CGPoint(x: view.center.x, y: 660)
+        button.setTitle(Constants.addToCartText, for: .normal)
+        button.titleLabel?.font = UIFont.systemFont(ofSize: 15, weight: .bold)
+        button.backgroundColor = UIColor(red: 0.07, green: 0.45, blue: 0.87, alpha: 0.90)
+        button.layer.cornerRadius = 10
+        return button
+    }()
+    
+    private lazy var deliverLabel: UILabel = {
+        let label = UILabel()
+        label.frame = CGRect(x: 40, y: 710, width: 250, height: 10)
+        label.text = Constants.deliveryText
+        label.font = UIFont.systemFont(ofSize: 12, weight: .bold)
+        label.textColor = .white
+        return label
+    }()
+    
+    private lazy var deliverDateLabel: UILabel = {
+        let label = UILabel()
+        label.frame = CGRect(x: 40, y: 725, width: 250, height: 10)
+        label.text = Constants.delivaryDateText
+        label.font = UIFont.systemFont(ofSize: 12, weight: .light)
+        label.textColor = .systemGray2
+        return label
+    }()
+    
+    private lazy var deliverVariantsLabel: UILabel = {
+        let label = UILabel()
+        label.frame = CGRect(x: 40, y: 740, width: 350, height: 10)
+        label.text = Constants.deliveryVariantsText
+        label.font = UIFont.systemFont(ofSize: 12, weight: .light)
+        label.textColor = .link
+        return label
+    }()
+    
+    private lazy var boxDeliveryImageView: UIImageView = {
+        let image = UIImageView()
+        image.image = UIImage(systemName: Constants.sistemBoxImageName)
+        image.frame = CGRect(x: 10, y: 710, width: 20, height: 20)
+        image.tintColor = .gray
+        return image
+    }()
+    
+    private lazy var howMuchTextField: UITextField = {
+        let field = UITextField()
+        field.frame = CGRect(x: 0, y: 0, width: 180, height: 20)
+        field.center = CGPoint(x: view.center.x, y: 610)
+        field.placeholder = Constants.howMuchPlaceholderText
+        field.font = UIFont.systemFont(ofSize: 15, weight: .medium)
+        field.textColor = .white
+        field.attributedPlaceholder = NSAttributedString(string: Constants.howMuchPlaceholderText, attributes: [NSAttributedString.Key.foregroundColor: UIColor.systemGray2])
+        field.textAlignment = .center
+        return field
+    }()
+    
+    // MARK: - Public properties
+    var productName = ""
+    var productImageName = [String]()
+    var productPrice = ""
```

Comment:
```suggestion
    var productPrice = ""
    
```
пробел между let и var